src/textual/validation.py

```diff
@@ -384,12 +384,15 @@ def __init__(
         minimum: int | None = None,
         maximum: int | None = None,
         failure_description: str | None = None,
+        strip_spaces: bool = False,
```

Comment:
I'm actually thinking this should default to `True`. I can't imagine why you would want to allow spaces.